### PR TITLE
[DE] Add `Maintenance`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
This PR proposes (together with #7928) to use the following terminology:

- `pull request` --> `Änderungsvorschlag`, `Änderungsanfrage` and `Änderungsantrag`
- `repositories` --> `Repositorien` or `Repositorys` 🤔 
- `merge` --> `zusammenführen` and ~~`fusionieren`~~ `Anfrage anwenden/akzeptieren`
- `fork` --> `Verzweigung`
- `branch` --> `Branch`, *perhaps* `Ast` (in any case this should not be too similar to `Abzweigung`)
- `commit` (`to commit`) - `Commit` (`in die Versionshistorie aufnehmen`)
- `contributor` --> `Mitwirkender` and `Beteiligter`

Some of these terms aren't used in this article. However, I'm currently working on translating the articles in `osu!_wiki/Contibution_guide` and then these will be essential. The articles themselves will come in later PRs.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!_wiki/Maintenance/de.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+Bedenke, dass beim Hinzufügen von Weiterleitungen für neue oder bereits existierende Artikel die Links prägnant und für den tatsächlichen Gebrauch konzipiert sein sollten.
+
+### CI-Checks
+
+Das Repository für das osu!-Wiki nutzt [kontinuierliche Integration](https://docs.github.com/en/actions/guides/about-continuous-integration) (auch *continuous integration* oder *CI* genannt), um eingegangene Änderungsanfragen automatisch auf typische Fehler zu überprüfen. Die Liste der Checks wird in der Datei [`continuous-integration.yml`](https://github.com/ppy/osu-wiki/blob/master/.github/workflows/continuous-integration.yml) konfiguriert.
+
+Die Datei [`package.json`](https://github.com/ppy/osu-wiki/blob/master/package.json) listet alle Plugins auf, die in der CI genutzt werden. Manche davon wurden von den Betreuern des osu!-Wikis entwickelt.
+
+Die CI-Checks laufen automatisch bei jedem Commit eines regelmäßigen Mitwirkenden. Damit die Änderungsanfrage fusioniert wird, wird von den Mitwirkenden erwartet, dass sie alle Fehler beheben, die von der CI aufgezeigt werden. Um den [Status der Überprüfungen](img/ci-status.png) zu sehen, mache das Folgende:
```
Comment (FamousPig) [inline wiki/osu!_wiki/Maintenance/de.md:50]:
```suggestion
Die CI-Checks laufen automatisch bei jedem Commit eines regelmäßigen Mitwirkenden. Damit die Änderungsanfrage akzeptiert wird, wird von den Mitwirkenden erwartet, dass sie alle Fehler beheben, die von der CI aufgezeigt werden. Um den [Status der Überprüfungen](img/ci-status.png) zu sehen, mache das Folgende:
```
```suggestion
Die CI-Checks laufen automatisch bei jedem Commit eines regelmäßigen Mitwirkenden. Damit die Änderungsanfrage zusammengeführt wird, wird von den Mitwirkenden erwartet, dass sie alle Fehler beheben, die von der CI aufgezeigt werden. Um den [Status der Überprüfungen](img/ci-status.png) zu sehen, mache das Folgende:
```
```suggestion
Die CI-Checks laufen automatisch bei jedem Commit eines regelmäßigen Mitwirkenden. Damit die Änderungsanfrage in den `master`-Branch aufgenommen wird, wird von den Mitwirkenden erwartet, dass sie alle Fehler beheben, die von der CI aufgezeigt werden. Um den [Status der Überprüfungen](img/ci-status.png) zu sehen, mache das Folgende:
```

File: wiki/osu!_wiki/Maintenance/de.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+## Administratoren
+
+*Hauptseite: [Liste der Verantwortlichen des osu!-Wikis](/wiki/osu!_wiki/Maintenance/List_of_maintainers)*
+
+Die Administratoren sind Personen mit [erhöhten Rechten](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/permission-levels-for-a-user-account-repository#collaborator-access-for-a-repository-owned-by-a-user-account) im [Repository `ppy/osu-wiki`](https://github.com/ppy/osu-wiki/), in dem alle Artikel und News-Beiträge gespeichert werden. Sie können Issues einstufen sowie kennzeichnen, offene Änderungsanfragen (Pull-Requests) verwalten und Entscheidungen über die Gegenwart und Zukunft des osu!-Wikis treffen.
+
+Verantwortliche führen eine finale Überprüfung von Änderungsvorschlägen durch und fusionieren diese. Wenn du bei einem Beitrag Hilfe benötigst, frage einen Betreuer im Kanal `#osu-wiki`.
```
Comment (FamousPig) [inline wiki/osu!_wiki/Maintenance/de.md:13]:
```suggestion
Verantwortliche führen eine finale Überprüfung von Änderungsvorschlägen durch und mergen diese. Wenn du bei einem Beitrag Hilfe benötigst, frage einen Betreuer im Kanal `#osu-wiki`.
```
```suggestion
Verantwortliche führen eine finale Überprüfung von Änderungsvorschlägen durch und führen diese mit dem Master-Branch zusammen. Wenn du bei einem Beitrag Hilfe benötigst, frage einen Betreuer im Kanal `#osu-wiki`.
```
```suggestion
Verantwortliche führen eine finale Überprüfung von Änderungsvorschlägen durch und akzeptieren diese. Wenn du bei einem Beitrag Hilfe benötigst, frage einen Betreuer im Kanal `#osu-wiki`.
```
```suggestion
Verantwortliche führen eine finale Überprüfung von Änderungsvorschlägen durch und wenden diese an. Wenn du bei einem Beitrag Hilfe benötigst, frage einen Betreuer im Kanal `#osu-wiki`.
```
Really don't like `fusioniert`. It feels way too exotic in this context.

File: wiki/osu!_wiki/Maintenance/de.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+### Erweiterung von unvollständigen Artikeln
+
+*Für einen möglichen Arbeitsumfang, siehe: [Liste von unvollständigen Artikeln (Englisch)](https://github.com/search?q=stub%3A+true+repo%3Appy%2Fosu-wiki+filename%3Aen.md)*
+
+Einige Artikel des osu!-Wikis sind unvollständig und es fehlen Informationen. Diese Artikel werden als *Stubs* gekennzeichnet, was heißt, dass sie wichtig genug sind, um als eigenständige Seite zu existieren, aber später vervollständigt werden. Trage dazu bei und teile dein Wissen, wenn du mit dem Artikelthema vertraut bist.
+
+### Querverlinkung
+
+Eine der Hauptmerkmale jedes Wikis ist *Vernetzung*, was bedeutet, dass Artikel auf verwandte Seiten verweisen, um dem Leser zu helfen, im Fluss zu bleiben. Füge für die Verbindung von Artikeln Links zu den genannten Begriffen hinzu, bei denen es für ein besseres Verständnis des Themas wichtig ist. Verweise bei Bedarf auf einzelne Abschnitte von Artikeln und nutze [Begriffsabgrenzungsseiten](/wiki/Article_styling_criteria/Formatting#artikel-für-die-begriffsabgrenzung) für mehrdeutige Begriffe.
```
Comment (FamousPig) [inline wiki/osu!_wiki/Maintenance/de.md:92]:
```suggestion
Eines der Hauptmerkmale jedes Wikis ist *Vernetzung*, was bedeutet, dass Artikel auf verwandte Seiten verweisen, um dem Leser zu helfen, im Fluss zu bleiben. Füge für die Verbindung von Artikeln Links zu den genannten Begriffen hinzu, bei denen es für ein besseres Verständnis des Themas wichtig ist. Verweise bei Bedarf auf einzelne Abschnitte von Artikeln und nutze [Begriffsabgrenzungsseiten](/wiki/Article_styling_criteria/Formatting#artikel-für-die-begriffsabgrenzung) für mehrdeutige Begriffe.
```

File: wiki/osu!_wiki/Maintenance/de.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+*Für eine Liste an Übersetzungen und ihre Vollständigkeit, siehe: [osu-wiki status](https://clayton.cc/osu-wiki-status/de)*
+
+Das osu!-Wiki wird von Menschen aus der ganzen Welt gelesen. Um deiner lokalen Community zu helfen und neue, großartige Spieler, Mapper, Modder und Entwickler in das Spiel zu locken, kannst du englische Artikel übersetzen oder existierende Übersetzungen aktualisieren, die in Rückstand geraten sind. Siehe dir die [Liste an Sprachen](/wiki/Article_styling_criteria/Formatting#locales) an, die vom osu!-Wiki unterstützt werden und stelle sicher, dass deine Übersetzung dem Prinzip der [Inhaltsparität](/wiki/Article_styling_criteria/Writing#inhaltsparität) folgt. Wenn du eine Sprache fließend sprechen kannst und ein erfahrener Schriftsteller bist, greife Schlüsselthemen auf wie Artikel über die [Regeln](https://github.com/ppy/osu-wiki/tree/master/wiki/Rules) oder die [Ranking-Kriterien](https://github.com/ppy/osu-wiki/tree/master/wiki/Ranking_Criteria). Falls du noch am Anfang mit deiner Schriftstellerkarriere stehst, dann wähle einen kleinen Artikel aus, um Hilfe und Betreuung von aktiven Kritikern zu erhalten.
```
Comment (FamousPig) [inline wiki/osu!_wiki/Maintenance/de.md:80]:
```suggestion
Das osu!-Wiki wird von Menschen aus der ganzen Welt gelesen. Um deiner lokalen Community zu helfen und neue, großartige Spieler, Mapper, Modder und Entwickler in das Spiel zu locken, kannst du englische Artikel übersetzen oder existierende Übersetzungen aktualisieren, die in Rückstand geraten sind. Siehe dir die [Liste an Sprachen](/wiki/Article_styling_criteria/Formatting#locales) an, die vom osu!-Wiki unterstützt werden und stelle sicher, dass deine Übersetzung dem Prinzip der [Inhaltsparität](/wiki/Article_styling_criteria/Writing#inhaltsparität) folgt. Wenn du eine Sprache fließend sprechen kannst und ein erfahrener Schriftsteller bist, greife Schlüsselthemen auf wie Artikel über die [Regeln](https://github.com/ppy/osu-wiki/tree/master/wiki/Rules) oder die [Ranking-Kriterien](https://github.com/ppy/osu-wiki/tree/master/wiki/Ranking_Criteria). Falls du noch am Anfang mit deiner Schriftstellerkarriere stehst, dann wähle einen kleinen Artikel aus, um Hilfe und Betreuung von aktiven Rezensenten zu erhalten.
```

`Kritiker` seems a little harsh.  Not a huge deal though.

**Review by FamousPig — CHANGES_REQUESTED** [review]

**Comment by The-Last-Cookie** [comment]
> Really don't like `fusioniert`. It feels way too exotic in this context.

Sure, I originally had this idea from the French wiki as they use the verb `fusionner`.

Another thing that I want to mention is that I used `Issue anlegst` instead of `Issue erstellst` in the last paragraph to have more word variety.

**Review by FamousPig — APPROVED** [review]

**Pull request merged**